### ISSUE TITLE
Spotify cannot shuffle album

### ISSUE THREAD

**Issue by rcandelu** [tizonia/tizonia-openmax-il#752]
Hello, 
Is it possible to play shuffle the album by album id?
Thanks
Regards,

**Comment by juanrubio** [comment]
Hi @rcandelu 

Have you tried adding ``-s|--shuffle``?
See also https://docs.tizonia.org/manual/global.html

**Comment by rcandelu** [comment]
Yes I tried -s and --shuffle but always start from the first track.
I run this command: 
tizonia --spotify-user $username --spotify-password $password --spotify-recover-lost-token --spotify-allow-explicit-tracks --spotify-album-id $albumid -s

Or

tizonia --spotify-user $username --spotify-password $password --spotify-recover-lost-token --spotify-allow-explicit-tracks --spotify-album-id $albumid --shuffle

But always start from the first track.
Tizonia version 0.22

**Comment by juanrubio** [comment]
@rcandelu 

You are absolutely right, I've just noticed now. This must be a regression. I'm marking it as a bug. The fix should be straightforward.

Thanks for reporting!.

**Comment by tirzak** [comment]
@juanrubio could you help me start?
